src/index.html

```diff
@@ -14,7 +14,7 @@ <h1 class="todo-header-text">today's to do</h1>
         <button id="header-btn"></button>
       </div>
       <form id="form" class="form" action="">
-        <input type="text" id="todo-input" placeholder="Add to your list" />
+        <input type="text" id="todo-input" placeholder="Add to your list" autofocus/>
         <button id="enter-input"></button>
       </form>
       <ul class="todo-list-wrap" id="todo-list-wrap"></ul>
```

Comment:
- [ ] [OPTIONAL] I think you mistakenly out `vw` instead of `vh` for the min-height property here :point_down: I would suggest to fix this as currently the todo is appearing at the very bottom of the page and is not visible to interact with unless you scroll way down :+1: 
```
body {
  margin: 0;
  padding: 0;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background-color: lightgray;
  min-height: 100vw;
}
```

Reply:
thanks for pointing that out Sir!

Reply:
fixed